python_graphql_client/graphql_client.py

```diff
@@ -69,9 +69,7 @@ async def execute_async(
 
         async with aiohttp.ClientSession() as session:
             async with session.post(
-                self.endpoint,
-                json=request_body,
-                headers=self.__request_headers(headers),
+                self.endpoint, json=request_body, headers={**self.headers, **headers},
```

Comment:
options should be passed here as well right ?

Reply:
aiohttp support `auth` argument. 
https://docs.aiohttp.org/en/stable/client_reference.html

Reply:
I didn't add the options parameter to the `exectue_async` method intentionally in this PR to limit the scope to configuring the `requests` library. To expand the scope to cover the `aiohttp`, I think we should consider that these two libraries can take options through the different parameter names. In case of `auth`, they use the same parameter name but `aiohttp` uses `veryfy_ssl` while `requests` uses `verify` for the same purpose. So, I'd like to tackle this separately after giving it some consideration unless you need this feature right now.